Src/Base/AMReX_Geometry.H

```diff
@@ -67,6 +67,32 @@ public:
     int coord;
 };
 
+    namespace detail {
+        template <typename T>
+        T bisect_prob_hi (amrex::Real plo, amrex::Real phi, amrex::Real idx, int ilo, int ihi, amrex::Real tol) {
+            T hi = static_cast<T>(phi - tol);
+            bool safe;
+            {
+                int i = int(Math::floor(hi - plo)*idx) + ilo;
```

Comment:
Should it be `floor((hi-plo)*idx)` instead of `floor(hi-pol)*idx`?

Reply:
Yes, good catch.